pkg/infra/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	DBConfig         defs.Database   `mapstructure:"db"`
 	HTTPConfig       HTTPConfig      `mapstructure:"http"`
 	Logging          LogConfig       `mapstructure:"logging"`
+	Commission       defs.Commission `mapstructure:"commission"`
```

Comment:
please apply this change on example yaml config. or wait for my changes with autogenerate for example config.